src/services/MobileService.ts

```diff
@@ -246,25 +247,18 @@ class MobileService {
         throw new Error('Client did not send the cipher text')
       }
 
-      // Remove any existing client backup shares for this backup method.
-      await this.prisma.clientBackupShare.deleteMany({
```

Comment:
If we had this function, how were we ending up with duplicates?

Reply:
Because the first commit was promoted to staging but not production, so duplicates were able to be created on staging